src/pyasm/search/sql.py

```diff
@@ -3444,29 +3444,29 @@ def get_database(self):
         return self.db_resource.get_database()
 
 
-    def add(self, name, type, length=None, not_null=False, primary_key=False):
+    def add(self, name, type, length=None, not_null=False, primary_key=False, default=None):
         if type == "text":
-            expr = self.impl.get_text(not_null=not_null)
+            expr = self.impl.get_text(not_null=not_null, default=default)
```

Comment:
You need to add the new kwarg to all of the DatabaseImpl classes. (ie. PostgreSQLImp, SQLServerImpl, etc.)

Reply:
I was unsure about other dbms. However, after consulting docs, I have added for other dbms as well.

Reply:
Please verify again.